okhttp-dnsoverhttps/src/test/java/okhttp3/dnsoverhttps/DnsOverHttpsTest.java

```diff
@@ -36,13 +37,15 @@
 import okio.ByteString;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
 import static java.util.Arrays.asList;
 import static java.util.Collections.singletonList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.fail;
 
+@ExtendWith(MockWebServerExtension.class)
```

Comment:
Need to register this manually in order for the junit5 android runner to pick it up.

Reply:
Can we set the system property to turn on ‘automatic’ extensions?

Reply:
Forgot about that

https://github.com/mannodermaus/android-junit5/issues/229

Also raised a question of O minimum, whch seems debateable.  https://github.com/mannodermaus/android-junit5/issues/228

Reply:
For follow up